website/docs/reference/dbt-jinja-functions/zip.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "zip"
+id: "zip"
+---
+
+### zip
+
+The `zip` context method can be used to used to return an iterator of tuples, where the i-th tuple contains the i-th element from each of the argument iterables. ([Python docs](https://docs.python.org/3/library/functions.html#zip))
+        :param 
```

Comment:
@jtcohen6 was this a typo?  or are these `:param` items meant to be there?  Glad to PR the change into the docs, just wanted to confirm first